cerberus/tests/test_validation.py

```diff
@@ -1830,3 +1830,50 @@ def test_contains(constraint):
         errors.MISSING_MEMBERS
     ].info[0]
     assert any(x in missing_actors for x in ('Eric Idle', 'Terry Gilliam'))
+
+
+def test_require_all_simple():
+    v = Validator({'foo': {'type': 'string'}}, require_all=True)
+    assert not v({})
```

Comment:
instead of assert statements you could use the helper functions [assert_fail](https://github.com/pyeve/cerberus/blob/master/cerberus/tests/__init__.py#L34) and [assert_success](https://github.com/pyeve/cerberus/blob/master/cerberus/tests/__init__.py#L72) that are more helpful for debugging. with the function that expects a failure one can also pass the expected error data, there are some uses of this as reference.